crates/wit-parser/src/ast/lex.rs

```diff
@@ -135,17 +135,6 @@ impl<'a> Tokenizer<'a> {
         Ok(ret)
     }
 
-    pub fn parse_str(&self, span: Span) -> Result<String> {
```

Comment:
@fibonacci1729 is this a function you'd like to keep around for now?

(if so @kulakowski-wasm would you be ok adding `#[allow(dead_code)]` here with a comment?)

Reply:
Yeah, i'll be using this in the near future.

Reply:
ofc happy either way!